content/starter-configs/tutorials/laravel.md

```diff
@@ -0,0 +1,153 @@
+`.tugboat/config.yml` in the git repository. Here's a basic configuration you can use as a starting point, with
+comments to explain what's going on:
+
+```yaml
+# Default Laravel 11 Tugboat starter config.
+# https://docs.tugboatqa.com/starter-configs/tutorials/laravel/
+services:
+  # What to call the service hosting the site.
+  webserver:
+    # This uses PHP 8.3.x with Apache: update to match your version of PHP.
+    image: tugboatqa/php:8.3-apache
+
+    # Set this as the default service. This does a few things
+    #   1. Clones the git repository into the service container
+    #   2. Exposes port 80 to the Tugboat HTTP proxy
+    #   3. Routes requests to the preview URL to this service
+    default: true
+    # Wait until the mysql service is done building.
+    depends: database
+
+    # A set of commands to run while building this service
+    commands:
+      # Commands that set up the basic preview infrastructure
+      init:
+        # Install opcache and mod-rewrite.
+        - docker-php-ext-install opcache
+        - a2enmod headers rewrite
+        # Install node
+        - apt-get update
+        - apt-get install -yq ca-certificates curl gnupg
+        - mkdir -p /etc/apt/keyrings
+        - curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o
+          /etc/apt/keyrings/nodesource.gpg
+        - echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x nodistro main" |
+          tee /etc/apt/sources.list.d/nodesource.list
+        - apt-get update
+        - apt-get install -yq nodejs
+        # Copy the tugboat .env we created before
+        - cp "${TUGBOAT_ROOT}/.tugboat/.env.tugboat" "${TUGBOAT_ROOT}/.env"
+        # Link the document root to the expected path. This example links
+        # /public to the docroot
+        - ln -snf "${TUGBOAT_ROOT}/public" "${DOCROOT}"
+        # Composer install
+        - composer install --optimize-autoloader
+        # Generate the random key if you didn't set APP_KEY before.
+        - php artisan key:generate
+        # Generate the db structure with some data.
+        - php artisan migrate --seed
+        # Compile vite resources
+        - npm install
+        - npm run build
+      update:
+        # Include the APP URL in .env
+        - echo "APP_URL=${TUGBOAT_DEFAULT_SERVICE_URL_HOST}" >> "${TUGBOAT_ROOT}/.env"
+        - composer install --optimize-autoloader
+        # Clear caches.
+        - php artisan config:cache
+        # Run any pending migrations.
+        - php artisan migrate --force
+        # Ensure storage permissions
+        - chgrp -R www-data "${TUGBOAT_ROOT}/storage"
+        - find "${TUGBOAT_ROOT}/storage" -type d -exec chmod 2775 {} \;
+        - find "${TUGBOAT_ROOT}/storage" -type f -exec chmod 0664 {} \;
+        # Compile vite templates
+        - npm install
+        - npm run build
```

Comment:
Since the `update` build phase only runs on base previews or previews with no base, I think you will want a `build` phase as well with some of these commands. For example, something like this? (would artisan migrate run on every deployment typically?)

```suggestion
        - npm run build
      build:
        - composer install --optimize-autoloader
        # Clear caches.
        - php artisan config:cache
        # Run any pending migrations.
        - php artisan migrate --force
        - npm install
        - npm run build
```

Reply:
Thanks! Yes, migrations should run on every deployment.